src/js/DevContest.js

```diff
@@ -72,9 +116,14 @@ $('#buttonRelease').click(function(error, result) {
 });
 
 $('#buttonRegister').click(function(error, result) {
-  DevContest.registerSubmission($('#submission').val(), {from: web3.eth.accounts[0]}, function(error, result) {
-    console.log(result);
-  });
+  if($('#validationCustom01').val().length != 0 && $('#validationCustom01').val() != 0 && $('#validationCustom01').val() != 0) {
```

Comment:
Don't understand why we need to check for the same condition twice, this could be refactored to:
`if($('#validationCustom01').val())`